app/components/ConfigItem.vue

```diff
@@ -147,7 +147,7 @@ const extraConfigs = computed(() => {
       <div v-if="config.ignores" flex="~ gap-2 items-start">
         <div i-ph-eye-closed-duotone my1 flex-none />
         <div flex="~ col gap-2">
-          <div v-if="!config.files">
+          <div v-if="Object.keys(config).some(configKey => configKey !== 'ignores' && configKey !== 'index') === false">
```

Comment:
There is also `name` - I don't know if there will be any others in the future.

Also, we need to display "invalid config" not just `v-else` as the ignores are not functional when other keys presents but not `files`

Reply:
Unless it's been changed in a recent release, `name` is also disallowed, something I discovered when building https://github.com/neostandard/neostandard/blob/bb370d2cec486eecd0f6057621b95628507eff5d/lib/main.js#L95-L100

Reply:
Are you sure the ignores are not functional when `files` is not there? As I commented here https://github.com/eslint/config-inspector/issues/80#issuecomment-2290972030 I think it is valid?

Reply:
I made the fix in https://github.com/humanwhocodes/config-array/pull/131

And which is now in https://github.com/eslint/rewrite/blob/e2a7ec809db20e638abbad250d105ddbde88a8d5/packages/config-array/src/config-array.js#L76

The fix only seems to apply to ESLint 9 tho

Reply:
It wasn't even called `@eslint/config-array` before v8. I don't think we could have the capability of getting that information from `@eslint/config-array` as we also need to "explains". I'd say they really should backport it to v8, as that is considered a bug. For config inspector, I'd say we better move forward to display based on the latest rule.

Reply:
@antfu Added a check for `name`

Reply:
> If we would rework FlatConfigItem from

I am up to it, we could do that in another PR